File: Java Barcode Scanner ME/app/src/main/java/de/dimedis/mobileentry/backend/response/SendHeartbeatResponseContent.java
package de.dimedis.mobileentry.backend.response;

import com.google.gson.annotations.SerializedName;

public class SendHeartbeatResponseContent extends BaseResponseContent {
    public static final String VERSIONS = "versions";

    @SerializedName(VERSIONS)
    private Versions mVersions;


    public Versions getVersions() {
        return mVersions;
    }

}

File: Kotlin Migration/NTMobileEntry/app/src/main/java/de/dimedis/mobileentry/backend/response/SendHeartbeatResponseContent.kt
package de.dimedis.mobileentry.backend.response

import com.google.gson.annotations.SerializedName

class SendHeartbeatResponseContent : BaseResponseContent() {
    companion object {
        const val VERSIONS = "versions"
    }

    @SerializedName(VERSIONS)
    private val mVersions: Versions? = null

    val versions: Versions?
        get() = mVersions
}